generator/config/query/mod.yaml

```diff
@@ -10,8 +10,34 @@ arguments:
     -
         name: divisor
         type:
-            - int
+            - number
```

Comment:
I suspected there were a few more that I missed previously. Thanks for finding those 🎉